{"name": "test_2", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 2", "trace": "def test_2():\n>       assert 1==2\nE       assert 1 == 2\n\ntest_case_01.py:7: AssertionError"}, "start": 1608626488470, "stop": 1608626488470, "uuid": "2024dad2-c922-4d80-ad4c-55744b9bed56", "historyId": "c776785f8cb7e66104e2cae03c9d3567", "testCaseId": "04cea479f7303fac9f7c52b86ba1de25", "fullName": "test_case_01#test_2", "labels": [{"name": "suite", "value": "test_case_01"}, {"name": "host", "value": "LAPTOP-T9T0JTJ0"}, {"name": "thread", "value": "2636-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case_01"}]}